patches/tModLoader/Terraria/ModLoader/CombinedHooks.cs

```diff
@@ -154,5 +154,58 @@ public static class CombinedHooks
 			}
 			return ret;
 		}
+
+		public static bool CanCaptureWith(Player player, Item item) {
+			bool? canCaptureWithOverall = null;
+			bool? canCaptureWithPlayer = PlayerLoader.CanCaptureWith(player, item);
+			bool? canCaptureWithItem = ItemLoader.CanCaptureWith(item, player);
```

Comment:
Should these be called after the early return checks?

Reply:
probably. will do this